### PR TITLE
Feat: rounded and responsive image added

### PR DESCRIPTION
**Pull request by anandsr21harshit**

### CODE REVIEW

**Review by varnita619 — APPROVED** [review]
Everything seems good

File: Components/Image/image.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    <link rel="stylesheet" href="./image.css">
+    <title>Image</title>
+</head>
+<body>
+    <div class="image-container">
+        <img src="./paris.jpg" alt="" class="round-image">
+    </div>
+    <div class="image-container">
+        <img src="./paris.jpg" alt="" class="responsive-image">
+    </div>
```
Comment (ddivyansh18) [inline Components/Image/image.html]: Add text to alt content of images

**Review by ddivyansh18 — CHANGES_REQUESTED** [review]

**Review by manish591 — COMMENTED** [review]
Looking good. Once you have added alt then ping me. I'll approve these changes.

File: Components/Image/image.html (hunk 1 above)
Comment (anandsr21harshit) [reply Components/Image/image.html]: Updated

**Review by manish591 — APPROVED** [review]
Good to merge

**Pull request merged**